debian/rules

```diff
@@ -9,6 +9,7 @@ override_dh_auto_configure:
 		-- \
 		-Dinstalled_tests=true \
 		-Ddefault_library=static \
+		-Dtests=unsafe
```

Comment:
This will be used on OBS so will run the unsafe tests on the OBS worker – I thought we wanted to avoid that?

Reply:
Oh, never mind me. Provided OBS doesn't build packages as root, which I'm reasonably sure it doesn't, they won't run because `-rfakeroot` is removed below so the `geteuid () == 0` check will fail.

Reply:
I was under the impression we wanted the unsafe test to be run by OBS during the autopkgtest step. Otherwise there's no point in adding the autopkgtest and the test would only be run manually by a determined developer (that is, almost never).

Reply:
As mentioned [before](https://github.com/endlessm/eos-payg/pull/41#discussion_r266817175), nothing runs autopkgtests at the moment.